test/stream.test.ts

```diff
@@ -213,6 +237,30 @@ describe('DataAndMoneyStream', function () {
       assert.equal(spy.callCount, count)
     })
 
+    it('should reject if the stream is in receiveOnly mode', async function () {
```

Comment:
Its probably good to duplicate this test in the opposite direction, client with receiveOnly=true.